docs/toc.json

```diff
@@ -71,6 +71,9 @@
       "title": "Region",
       "type": "compute/region"
     }, {
+      "title": "Service",
+      "type": "compute/service"
+    }, {
```

Comment:
Oh weird, I don't see Service in the navigation. `implemented` went away. We version <kbd>toc.json</kbd> in the same fashion as the rest of the JSON/markdown.